src/main/java/bio/terra/service/dataset/flight/ingest/IngestValidateGcpRefsStep.java

```diff
@@ -35,13 +36,16 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
     // Then probe the file system to validate that the file exists and is part
     // of this dataset. We check all ids and return one complete error.
 
-    Set<String> invalidRefIds = new HashSet<>();
+    Set<InvalidRefId> invalidRefIds = new HashSet<>();
     for (Column column : table.getColumns()) {
       if (column.isFileOrDirRef()) {
         List<String> refIdArray = bigQueryPdao.getRefIds(dataset, stagingTableName, column);
         List<String> badRefIds = fileDao.validateRefIds(dataset, refIdArray);
         if (badRefIds != null) {
```

Comment:
I don't see how the result of `validateRefIds()` can be `null`. (In general, we shouldn't have functions that return a collection that can also return null.)

Reply:
I don't think it can be null either. Will change.